### ISSUE TITLE
Question regarding Server skipping placing parts from a deleted network portion

### ISSUE THREAD

**Issue by yujukitten** [CyclopsMC/IntegratedDynamics#461]
#### Issue type:

_Only keep the issue types that are applicable._

- :bug: Bug  ?
- :snail: Performance issue  ?
- :question: Question

Question :  How do I get rid of this error and or prevent it from happening again when moving logic cables etc?

____
#### Short description:
I deleted part of a network on my server via the dismantling tool but now I am receiving errors in the server logs

#### Expected behaviour:

To be able to remove the portion of network and move it to a new location

#### Actual behaviour:
It deleted but now Receiving error messages that their is no path. ( I am also experiencing massive lag but this could be purely coincidental)

#### Steps to reproduce the problem:  ( Not sure if this is applicable as in away i am more asking a question about this error ) 

____
#### Versions:

_Exact versions of the following mods, not just *latest*._

- This mod: 1.12.2.0.10.0
- CyclopsCore: 1.12.2.0.11.0
- Minecraft: 1.12.2
- Forge: 14.23.1.2560

#### Server errors :  ( I would post the log but I am not sure where to actually find this information since I do not crash Sorry ) 

21.12 21:07:02 [Server] INFO [21:07:02] [Server thread/WARN]: Can't keep up! Did the system time change, or is the server overloaded? Running 5622ms behind, skipping 112 tick(s)
21.12 21:07:01 [Server] INFO [21:07:01] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-459, y=87, z=1023} in world 0 because it has no valid path element.
21.12 21:07:01 [Server] INFO [21:07:01] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-458, y=87, z=1023} in world 0 because it has no valid path element.
21.12 21:07:01 [Server] INFO [21:07:01] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-457, y=87, z=1023} in world 0 because it has no valid path element.
21.12 21:07:01 [Server] INFO [21:07:01] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-460, y=86, z=1022} in world 0 because it has no valid path element.
21.12 21:07:01 [Server] INFO [21:07:01] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-459, y=86, z=1022} in world 0 because it has no valid path element.
21.12 21:06:58 [Server] INFO [21:06:58] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-458, y=86, z=1022} in world 0 because it has no valid path element.
21.12 21:06:58 [Server] INFO [21:06:58] [Server thread/WARN] [Integrated Dynamics]: Skipped loading part from a network at position BlockPos{x=-460, y=87, z=1023} in world 0 because it has no valid path element.

_Don't paste contents here directly, but use something like http://pastebin.com/_

**Comment by josephcsible** [comment]
What's the "dismantling tool"?

**Comment by yujukitten** [comment]
the wrench that comes with maybe integrated tunnels ? shift_right click to easily remove the logic cables, importers, exporters and interfaces. May even work with batteries, variable stores etc but I have never tried ?

**Comment by rubensworks** [comment]
The lag is unrelated. On the contrary, you should even experience less lag because of this :-)
Also, this is not an error, just a warning.

The wrench should work fine for removing cables. Perhaps something broke recently....

@yujukitten Could you try opening a clean world, placing some cables (with parts on them), removing them using the wrench, and seeing if you get the same warning again after a restart?

**Comment by yujukitten** [comment]
I can certainly try that on a local single player easier is this captured to a log in single player by chance ?

Thank you for clarifying its just a warning *^^* so I can stop trying to directly "repair" it 


Ok I recreated The Auto Terracotta setup I had on a brand new Local Single player Garden of glass world.
Everything was the same except originally sand was being fed from a storage drawer, that was fed from refined storage. but it does not  appear it got repeated. (But I definitely agree it could have been caused by something else breaking since my Log file for yesterday is 8 gbs ... and I would never have noticed that had I not gone to try this so Thank you again. )

I will include the log file I checked in case I am looking in the wrong place. 

[fml-client-latest.log](https://github.com/CyclopsMC/IntegratedDynamics/files/1583378/fml-client-latest.log)
I have included an image of the setup but I think seriously the first belief that something else broke and caused the warning is most likely the culprit.
![2017-12-22_11 56 56](https://user-images.githubusercontent.com/25888250/34311095-abe2c2ca-e718-11e7-94b2-b993da64f70d.png)
